project/cmake/modules/FindLibDvd.cmake

```diff
@@ -4,154 +4,191 @@ if(ENABLE_DVDCSS)
 endif()
 
 if(NOT WIN32)
-  foreach(dvdlib ${dvdlibs})
-    file(GLOB VERSION_FILE ${CORE_SOURCE_DIR}/tools/depends/target/${dvdlib}/DVD*-VERSION)
-    file(STRINGS ${VERSION_FILE} VER)
-    string(REGEX MATCH "VERSION=[^ ]*$.*" ${dvdlib}_VER "${VER}")
-    list(GET ${dvdlib}_VER 0 ${dvdlib}_VER)
-    string(SUBSTRING "${${dvdlib}_VER}" 8 -1 ${dvdlib}_VER)
-    string(REGEX MATCH "BASE_URL=([^ ]*)" ${dvdlib}_BASE_URL "${VER}")
-    list(GET ${dvdlib}_BASE_URL 0 ${dvdlib}_BASE_URL)
-    string(SUBSTRING "${${dvdlib}_BASE_URL}" 9 -1 ${dvdlib}_BASE_URL)
-    string(TOUPPER ${dvdlib} DVDLIB)
-
-    # allow user to override the download URL with a local tarball
-    # needed for offline build envs
-    # allow upper and lowercase var name
-    if(${dvdlib}_URL)
-      set(${DVDLIB}_URL ${${dvdlib}_URL})
+  if(CMAKE_CROSSCOMPILING)
+    if(PKG_CONFIG_FOUND)
+      pkg_check_modules(PC_DVD ${dvdlibs} QUIET)
+      #pkg_check_modules(PC_DVD libdvdread libdvdnav libdvdcss)
     endif()
-    if(${DVDLIB}_URL)
-      get_filename_component(${DVDLIB}_URL "${${DVDLIB}_URL}" ABSOLUTE)
-    else()
-      set(${DVDLIB}_URL ${${dvdlib}_BASE_URL}/archive/${${dvdlib}_VER}.tar.gz)
+ 
+    find_path(LIBDVD_INCLUDE_DIR dvdcss/dvdcss.h PATHS ${PC_DVD_INCLUDE_DIRS})
+    find_library(DVDREAD_LIBRARY NAMES dvdread libdvdread PATHS ${PC_DVD_libdvdread_LIBDIR})
+    find_library(DVDNAV_LIBRARY NAMES dvdnav libdvdnav PATHS ${PC_DVD_libdvdnav_LIBDIR})
+    find_library(DVDCSS_LIBRARY NAMES dvdcss libdvdcss PATHS ${PC_DVD_libdvdcss_LIBDIR})
+
+    include(FindPackageHandleStandardArgs)
+    find_package_handle_standard_args(LIBDVD REQUIRED_VARS LIBDVD_INCLUDE_DIR DVDREAD_LIBRARY DVDNAV_LIBRARY DVDCSS_LIBRARY)
+    if(LIBDVD_FOUND)
+      set(LIBDVD_LIBRARIES ${DVDREAD_LIBRARY} ${DVDNAV_LIBRARY} ${DVDREAD_LIBRARY})
```

Comment:
making this line
```
set(LIBDVD_LIBRARIES ${DVDREAD_LIBRARY} ${DVDCSS_LIBRARY})
```
allows it to compile for me. seems the double ${DVDNAV_LIBRARY} in the `core_link_library` causes problems.